Containers/nextcloud/cron.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
-set -eu
-
-while true; do
-    php -f /var/www/html/cron.php &
-    sleep 5m
-done
```

Comment:
but one thing, why is there a "&"?

Reply:
Because it shall spawn a new process every 5min (even if they run in parallel then)

Reply:
I don't understand why the & changes the logic, beside running both commands at the same time

Reply:
Because the command would be blocking otherwise so onky after it being finished counting the 5min and then starting the next process iirc

Reply:
so you mean that in the time of the five minutes the time cron.php needs to run is ignored?

Reply:
Yes

Reply:
but is there not a `& wait` missing? since if there is none them php command would be stopped?

Reply:
What would be the advantage? Can you propose a PR?

Reply:
I see. Can you propose a PR?

Reply:
can you please do it? i don't like to create prs with my mobile phone

Reply:
https://github.com/nextcloud/all-in-one/pull/2974